pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -230,6 +230,9 @@ def tearDown(self):
         del self.G
         del self.G1
 
+    def test__eq__(self):
+        model_copy = self.G1.copy()
+        self.assertTrue(G1.__eq__(model_copy))
```

Comment:
@Vaibhavdixit02 Test are failing for this line `self.assertTrue(G1.__eq__(model_copy))`, please fix it to self.assertTrue(self.G1.__eq__(model_copy))`also add test cases when they are not equal.

Reply:
Fixed and added.